### DISCUSSION TITLE
Disable null `sortKey` and none sortDirection in a sortable DataTable

### DISCUSSION THREAD

**Discussion by ibilux** [carbon-design-system/carbon-components-svelte#1746] [Q&A]
Is there a way to disable the null `sortKey` and none `sortDirection` in a sortable DataTable, and use only `descending` and `ascending` values ?

**Comment by metonym** [comment]
Released in [v0.102.0](https://github.com/carbon-design-system/carbon-components-svelte/releases/tag/v0.102.0)

New `sortAlways` prop, with per-column support. If `true`, only sorts ascending or descending.

Examples:
- https://svelte.carbondesignsystem.com/components/DataTable#sortable-sort-always
- https://svelte.carbondesignsystem.com/components/DataTable#sortable-per-column-sortalways
